app/models/education_navigation_ab_test_request.rb

```diff
@@ -8,7 +8,7 @@ class EducationNavigationAbTestRequest
   def initialize(request, content_item)
     @content_item = content_item
     @ab_test = GovukAbTesting::AbTest.new("EducationNavigation", dimension: 41)
-    @requested_variant = @ab_test.requested_variant request
+    @requested_variant = @ab_test.requested_variant(request.headers)
```

Comment:
this does not the govuk_ab_testing, i.e. why pass the headers as opposed to the request as [shown in the docs](https://github.com/alphagov/govuk_ab_testing#usage)